packages/viewer/src/modules/converter/Converter.js

```diff
@@ -186,7 +202,19 @@ export default class Coverter {
       const type = this.getSpeckleType(obj)
       if (this[`${type}ToBufferGeometry`]) {
         return await this[`${type}ToBufferGeometry`](obj.data || obj, scale)
-      } else return null
+      }
+      /**
+       * Regarding #723. This would be more generic and possibly handle other
+       * types with missing direct convertor, however I don't feel it is the
+       * 'convert' fuction's place to handle this...
+       */
```

Comment:
Agreed. The whole traversal & conversion logic needs to be cleaned up massively anyway...